src/__test__/utils/Cluster.ts

```diff
@@ -295,11 +293,65 @@ export class Cluster {
     await writeFile(this.path("./docker-compose.yaml"), stringify(config));
   };
 
+  private healthy = async (...nodes: string[]) => {
+    nodes = !nodes.length
+      ? Array.from({ length: this.count }, (_, i) => `esdb-node-${i}`)
+      : nodes;
+
+    const healthy = new Set();
+
+    while (healthy.size !== nodes.length) {
+      for (const node of nodes) {
+        if (healthy.has(node)) continue;
+
+        try {
+          const response = await exec(
+            node,
+            `curl --fail --insecure http${
+              this.insecure ? "" : "s"
+            }://localhost:2113/health/live`,
+            { cwd: this.path() }
+          );
+          // console.log(
```

Comment:
**QUESTION:** Do we want to leave this commented out `console.log` (and others in this file)?